pages/bloodline.php

```diff
@@ -93,31 +93,31 @@ function bloodline() {
 			'ratio_multiplier' => 100
 		),
 		'ninjutsu_boost' => array(
-			'text' => "[BL_SKILL] * [RATIO] -> <span class='amount'>[AMOUNT] extra ninjutsu offense (+[PERCENTAGE]%)</span>"
+			'text' => "[BL_SKILL] * [RATIO] -> <span class='amount'>[AMOUNT] extra Ninjutsu offense</span><br><span style='padding-left: 15px; font-size: smaller; font-style: italic'>Boost decreases to 75% strength as Bloodline Skill exceeds Ninjutsu skill</span>"
```

Comment:
Nice-to-have in a later follow up would be an actual value here shown, e.g. if player's BL skill is greater than Nin skill we add a `(Power currently reduced by 17% due to Bloodline skill exceeding Ninjutsu skill)` message, but for players whose BL skill is less than offense, we show nothing

Reply:
I like it! Taking note for when we update the bloodline page, it's one of the higher priority to replace IMO.